pkg/console/util.go

```diff
@@ -581,12 +629,10 @@ func doUpgrade(g *gocui.Gui) error {
 	}
 
 	scanner := bufio.NewScanner(stdout)
-	scanner.Split(bufio.ScanLines)
 	for scanner.Scan() {
 		printToPanel(g, scanner.Text(), upgradePanel)
 	}
 	scanner = bufio.NewScanner(stderr)
-	scanner.Split(bufio.ScanLines)
```

Comment:
BTW, when using `bufio.NewScanner`, the `bufio.ScanLines` is already split function by default. We don't need to set it up again.